src/main/java/com/gamesage/store/service/UserService.java

```
@@ -62,6 +63,10 @@ public List<User> findAll() {
     }
 
     public User createOne(User userToAdd) {
+        Optional<User> alreadyExistedUser = repository.findByLogin(userToAdd.getLogin());
```

Why this was added? If a user with the login already exists, it should throw an error.